apps/backoffice-v2/src/lib/react-query/query-client.ts

```diff
@@ -27,24 +33,27 @@ export const queryClient = new QueryClient({
     },
   },
   queryCache: new QueryCache({
-    onError: async error => {
-      if (isErrorWithCode(error) && error.code === 401) {
-        const authenticatedUser = authQueryKeys.authenticatedUser();
-
-        void queryClient.cancelQueries();
-        queryClient.setQueryData(authenticatedUser.queryKey, {
-          user: undefined,
-        });
-      }
-
+    onError: (error: unknown) => {
```

Comment:
My previous comment was resolved but the change have not been made. 🙂 
```suggestion
    onError: (error) => {
```